builds/android/res/values/strings.xml

```diff
@@ -20,4 +20,5 @@
 	<string name="thai">Thai</string>
 	<string name="vietnamese">Vietnamese</string>
 	<string name="cancel">Cancel</string>
-</resources>
+	<string name="no_game_found">No game found in $PATH</string>
```

Comment:
games has been replaced with game here. Could you push another commit to master adding the missing "s"?